spec/model/relation_spec.rb

```diff
@@ -159,6 +159,39 @@ class User < Foo::Model; end
       @user = Foo::User.baz.first
       expect(@user.id).to eq(4)
     end
+
+    context 'when there is another model present' do
```

Comment:
Agree w/ the nature of these specs. Is it possible to combine them into one spec and narrow the coverage. How about?

```ruby
before do
  Her::API.setup url: "https://api.example.com" do |builder|
    builder.adapter :test do |stub|
      # ...
      stub.get("/users?where=6") { ok! [{ id: 4, fullname: "Tobias Fünke" }] }
      stub.get("/users?where=7") { ok! [{ id: 2, fullname: "Lindsay Fünke" }] }
      stub.get("/users") { ok! [{ id: 1, fullname: "Tobias Fünke" }] }
    end
  end

  spawn_model "Foo::User" do
    # ...
    scope :baz, -> { bar(6) }
  end

  spawn_model "Bar::User" do
    scope :baz, -> { where(where: 7) }
  end
end

# ...

it "does not share scope with other models" do
  expect(Bar::User.scoped).not_to respond_to(:foo, :bar)
  expect(Foo::User.scoped.baz.params[:where]).to eq(6)
  expect(Bar::User.scoped.baz.params[:where]).to eq(7)
end
```

Reply:
Combined. 👍